mac

```diff
@@ -144,61 +143,31 @@ brew "coreutils"
 brew "yarn"
 cask "gpg-suite"
 
+# Spark recommended stuff
+brew "postman"
+brew "slack" 
+brew "google-chrome"
+brew "dashlane"
+brew "psequel"
+brew "iterm2"
+brew "dashlane"
```

Comment:
Double up on yourself here with two dashlanes. Perhaps putting these in alphabetical order would be a plus.